README.md

```diff
@@ -60,9 +56,11 @@ Set these mandatory options in the rails_helper:
 
 | Option | Value | Description |
 | -- | -- | -- |
-| header_file_path | Pathname instance or fullpath string | Markdown file that will be included at the top of the documentation. It should contain title and some basic info about the api. |
+| body_file_path | Pathname instance or fullpath string | Json file that will include openapi version, the basic information about the api, api title and api version. |
 | desc_folder_path | Pathname instance or fullpath string | Folder with markdown descriptions. |
-
+| schema_request_folder_path | Pathname instance or fullpath string | Folder with request schemas of resources. |
+| schema_response_folder_path | Pathname instance or fullpath string | Folder with response schemas of resources. |
+| schema_response_fail_file_path | Pathname instance or fullpath string | Json file that with the default schema of a failed response. |
```

Comment:
I don't think we should have a config for this, because it's implicit.
@d4be4st @vr4b4c